### PR TITLE
Fix script for Ghidra 10.3.1

### PR DESCRIPTION
**Pull request by KieronJ**
Replace removed createPointer with applyData.
Replace removed Scalar.subtract with getValue -> getLong(0).

Using applyData also prevents exceptions when a pointer is already defined.

### CODE REVIEW

File: SonyPSPResolveNIDs.py [hunk 1]
```diff
@@ -296,13 +297,13 @@ def getModuleInfoAddrFromLoadCommands():
     # 2nd component is p_offset
     load_offset = loadcmd.getComponent(1).getValue().getValue() # Data->Scalar->Long
```
Comment (Nemoumbra) [inline SonyPSPResolveNIDs.py]: What do you think about using 
`load_offset = loadcmd.getComponent(1).getLong(0)`?

The same can be applied to the 299th line as well (make it `load_paddr = loadcmd.getComponent(3).getLong(0)`), I think.

**Review by Nemoumbra — COMMENTED** [review]
I personally don't like the overall script structure, but it's not that bad. I absolutely don't think that you have to fix these problems, but while we're at it, maybe you can fix the typo at the line 122 (it should be `.lib.ent` there, not `.lib.stub`, someone did a little ctrl+C, ctrl+V)?

File: SonyPSPResolveNIDs.py (hunk 1 above)
Comment (KieronJ) [reply SonyPSPResolveNIDs.py]: That seems to work, looks better too.

**Comment by KieronJ** [comment]
> maybe you can fix the typo at the line 122 (it should be .lib.ent there, not .lib.stub, someone did a little ctrl+C, ctrl+V)?

Fixed now.

**Review by John-K — APPROVED** [review]
Looks great, thanks for this. I haven't had time to dig into any of the docs to figure out how to fix this.

**Pull request merged**